src/manifest.json

```diff
@@ -9,11 +9,7 @@
   "content_scripts": [
     {
       "matches": [
-        "https://github.com/*",
-        "https://gitlab.com/*",
-        "https://bitbucket.org/*",
-        "https://*.gogs.io/*",
-        "https://*.gitea.io/*"
+        "*://*/*"
```

Comment:
I didn't want to match all urls, either. However, regex patterns aren't allowed in the hostname portion of the url (which is relevant for GitLab, which is usually used in individual deployments rather than just gitlab.com). Here's the relevant doc: https://developer.chrome.com/extensions/match_patterns

Reply:
For example, here's a url for gitlab which cannot be matched by the allowed match patterns: `gitlab.example.com` Wildcards for the subdomains of a domain are allowed, but specifying a subdomain and wildcarding the domain is not allowed. According to Chrome, you either wildcard the entire hostname or just the subdomain.